pipetty.c

```diff
@@ -103,7 +103,7 @@ int main(int argc, const char **argv)
             dup2(p[0], 0);
             close(p[0]);
 
-            execlp("less", "less", "-R", "-", 0);
+            execlp("less", "less", "-R", "-", NULL);
```

Comment:
A compiler insisting on writing `0` as `NULL`?  Without even a semantic change, like `nullptr`... That's weird.

Reply:
At least gcc 7.5 recommends it. Thanks for merging :-)

Reply:
Ah, `-Wall` got lost when I converted to CMake, thanks for pointing out the warning.